### DISCUSSION TITLE
Remote Play

### DISCUSSION THREAD

**Discussion by Laveniuk** [EmulatorJS/EmulatorJS#415] [Q&A]
Hello!
I have seen remote play on an old version on this JS emulator.  Is that still a thing?
How do i set it up!?

**Comment by ethanaobrien** [comment]
Netplay is not supported at the moment. There were several big bugs that I scrapped the whole thing out. This is planned to eventually be supported at some point but not at the moment

**Reply by Laveniuk** [reply]
Oh shoot!  I seen it being used on retrogames.cc
What are they using then?
![image](https://user-images.githubusercontent.com/27043687/201395583-07d56de8-dbb1-4fac-8e61-f012cd722b9c.png)
![image](https://user-images.githubusercontent.com/27043687/201395647-c080f9d1-38c5-4853-b63f-2f713574baa6.png)

**Reply by ethanaobrien** [reply]
They are using emulatorjs.com  if you choose to use this - please note, I cannot help on any issues you may encounter there and note that that project normally uses 5-10 times more cpu, along with choppier audio, even on high end devices. There are also ads and several features completely unavailable

**Reply by Laveniuk** [reply]
Ok!  Are they two different products then?
I was able to get that .com emulator working quite easy, however it seems to have an issue with mapping controller buttons!  
As you said you cannot help with that so i will be using yours!  At least you can play games with it! :)  LOL
I cannot wait to see you get Netplay working!

**Reply by Laveniuk** [reply]
Ok, i think i have your emulator running, but I am now having controller mapping issues.
Is it my controllers?!
I can map Select/Start/A/B   It just doesnt register a button press...

http://dadbodhub.com/retro-games/

**Reply by ethanaobrien** [reply]
That is weird. I'm not sure whats wrong there, and your the second person thats reported that. Could you see if there is anything in the console?

**Reply by Laveniuk** [reply]
I would love to help, but you can consider me dumb when i comes to this stuff....  Where is the console?

**Reply by ethanaobrien** [reply]
It's all good. Just right click the screen > click "inspect" > then click the "console" tab at the top to open the console.

Please set `EJS_DEBUG_XX` to true in your scripts and then reload the page and try to map and use a button on a gamepad. The in the console, right click and click "save as" and then could you upload and send the log to this GitHub issue

**Reply by Laveniuk** [reply]
![image](https://user-images.githubusercontent.com/27043687/201485385-bfccc038-f43a-40c6-95db-6be2ebbec7e7.png)

Is that correct?

**Reply by Laveniuk** [reply]
You know what!  I don't think you have to worry about it!
I just tried again in Chrome and Edgeand it works fine!   There must be something messy in the Brave Browser.

**Reply by ethanaobrien** [reply]
Oh that's weird, thanks for the information

Ps: the t on the true should be lowercase

**Reply by Laveniuk** [reply]
I dont know why i need to know this now.....  But im still not seeing this debug log!  
Just trying to learn more about this stuff i guess!   
Ive changed the script to be 
![image](https://user-images.githubusercontent.com/27043687/201485940-ac8eb819-1980-48e7-bd4a-c8af812a2de9.png)

**Reply by ethanaobrien** [reply]
It should be in the console, but if this is only reproducible on brave then, as you said, it's probably a problem with a speaker. So don't worry about getting me anything else. Thank you!

**Reply by Laveniuk** [reply]
Yeah!  It works fine in Chrome and Edge!
Thanks man!   Can't wait to see Remote play!!! Let me know if you need help with anything!  I can do my best to try!
